File: app/src/main/java/in/iot/lab/bitscan/interfaces/IScanner.java
package in.iot.lab.bitscan.interfaces;

import android.graphics.Bitmap;



import in.iot.lab.bitscan.enums.ScanHint;

/**
 * Interface between activity and surface view
 */

public interface IScanner {



    void displayHint(ScanHint scanHint);

    void onPictureClicked(Bitmap bitmap);
}

File: app/src/main/java/in/iot/lab/bitscan/interfaces/IScanner.kt
package `in`.iot.lab.bitscan.interfaces

import `in`.iot.lab.bitscan.enums.ScanHint
import android.graphics.Bitmap

/**
 * Interface between activity and surface view
 */
interface IScanner {
    fun displayHint(scanHint: ScanHint?)
    fun onPictureClicked(bitmap: Bitmap?)
}